specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1882,15 +1884,23 @@
                     }
                 },
                 "allowedSourceAddressPrefix": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefixes\" field. should be IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""
+                },
+                "allowedSourceAddressPrefixes": {
+                    "type": "array",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefix\" field.",
+                    "items": {
+                        "type": "string",
+                        "description": "IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""

```

Comment:
Some grammar and typo correction:
`"IP address or CIDR, for example \"192.168.0.3\" or \"192.168.0.0/16\"."`

Reply:
Resolved